### PR TITLE
journal, removes trailing slashes from robots disallow paths

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: pillar/journal-public.sls [hunk 1]
```diff
@@ -20,15 +20,18 @@ journal:
             User-Agent: *
             Disallow: $robots_disallow
             Disallow: /download/
+        # regarding trailing slashes:
+        # - https://developers.google.com/search/reference/robots_txt#url-matching-based-on-path-values
+        # it shouldn't make a difference but turnitinbot thinks it does
```
Comment (thewilkybarkid) [inline pillar/journal-public.sls]: Which bit are you looking at? I see:

Screenshot 2020-02-18 at 08 36 46

Where it does make a difference (which makes sense, as they are different paths).

Comment (lsh-0) [reply pillar/journal-public.sls]: these bits:

Screenshot at 2020-02-20 09-49-08
Screenshot at 2020-02-20 09-52-01
Screenshot at 2020-02-20 09-52-15

see this:
Screenshot at 2020-02-20 09-53-36

My understanding all this time has been that these robots.txt rules were to block the search results from being scraped. Was that correct? 

I've only just now looked at how we do pagination with the search results (assuming it was `/search/something`) and it's actually `/search?something`, which means the turnitbot guys are right and I was wrong - blocking `/search/` would *not* have included `/search`, which is what we're after (if my understanding of our intent is correct).

Comment (thewilkybarkid) [reply pillar/journal-public.sls]: That’s my understanding too, so the change is right. Let’s drop the comment though, since that isn’t.

Comment (lsh-0) [reply pillar/journal-public.sls]: done

**Pull request merged**